src/s2wasm.h

```diff
@@ -1193,6 +1198,42 @@ class S2WasmBuilder {
         }
       }
```

Comment:
Oh, that's creating the `_start` function, creating enough locals to match `main`'s (or rather, whatever `startFunction` says the function name is) expected arguments, and calling `main` with these arguments. It then exports `_start` and creates the `(start $_start)` entry.